### PR TITLE
Added What is a Project Page

### PR DESCRIPTION
**Pull request by johnbhiggins7**
### Description
Added page "What is A Project"

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-XXX  

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Removed outdated references

-----
<a href="https://stackblitz.com/~/github.com/flutterflow/flutterflow-documentation/tree/what-is-a-project-page">Review PR in StackBlitz</a> _Submitted with [StackBlitz](https://stackblitz.com/~/github.com/flutterflow/flutterflow-documentation/tree/what-is-a-project-page)._

### CODE REVIEW

File: docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (leighajarett) [inline docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md:1]: Can we move this page into a different sidebar location? "FlutterFlow Resources" or something like that? I dont think this section belongs in the "Understanding the FlutterFlow UI" section

File: docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+slug: how-to-create-find-organize-projects
+title: How to Create, Find, and Organize Projects
+tags: []
+toc_max_heading_level: 5
+---
+# How to Create a Project
```
Comment (leighajarett) [inline docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md:7]: @johnbhiggins7 do you think we should be including screenshots or recording arcade's for these? I feel like we need something visual here

Comment (johnbhiggins7) [reply docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md:7]: @leighajarett totally. I was adding all the text first and then going to add in the visuals next.

File: docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md (hunk 1 above)
Comment (johnbhiggins7) [reply docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md:1]: Yep. This was all done before we had figured out that "Resources was deleted"

**Comment by PoojaB26** [comment]
Closing this as #14 has these changes.

**Pull request closed without merging**